converter/output/rita/dbmanager.go

```diff
@@ -32,18 +21,27 @@ type DBMetaInfo struct {
 	AnalyzeVersion string        `bson:"analyze_version"` // Rita version at analyze
 }
 
-//OutputDB wraps a *mgo.Session connected to MongoDB
```

Comment:
Unfortunately, this class used to be named OutputDB even though it managed several MongoDB databases. It's main use was to hand out new mgo.Collection instances connected to conn collections in different databases. Now it hands out rita.DB instances.